src/BuiltInTools/dotnet-watch/Program.cs

```diff
@@ -382,19 +378,5 @@ public void Dispose()
             _console.CancelKeyPress -= OnCancelKeyPress;
             _cts.Dispose();
         }
-
-        private static void RegisterAssemblyResolutionEvents(string sdkRootDirectory)
```

Comment:
Could we leave the dotnet-watch changes as-is. AFAIK, it's not being cross-gened today.

Reply:
Sure -- done.